### ISSUE TITLE
DialogSlot preventing Admin views from loading

### ISSUE THREAD

**Issue by mr337** [lektor/lektor#147]
I'm getting an issue with Lektor 1.2.1. I am not able to view any admin views. Upon doing so receiving this on console 

```
ERROR in ./js/views/App.jsx
Module not found: Error: Cannot resolve 'file' or 'directory' ../components/DialogSlot in /home/mr337/Dev/blog/.venv/lib/python2.7/site-packages/lektor/admin/static/js/views
 @ ./js/views/App.jsx 16:17-52

```

And this in FF console

```
Error: Cannot find module "../components/DialogSlot"
...ction webpackMissingModule() { var e = new Error("Cannot find module \"../compon...app.js ... > eval (line 16, col 82)
```

Finding it odd things were working I decided to take a peek in `js/components`

```
BaseComponent.jsx  BreadCrumbs.jsx  Component.jsx  dialogSlot.jsx  Link.jsx  RecordComponent.jsx  RecordEditComponent.jsx  ServerStatus.jsx  Sidebar.jsx  SlideDialog.jsx  ToggleGroup.jsx
```

Looks like `DialogSlot.jsx` got renamed to `dialogSlot.jsx`. Updating the filename in App.jsx fixes the issue. 

What is odd is master is shows `DialogSlot.jsx` while my Lektor install via pip is showing `dialogSlot.jsx`?

**Comment by mitsuhiko** [comment]
Not sure why that file would have a lowercase. It's named `DialogSlot.jsx` anywhere I can find it. That includes the tarball on pypi.

**Comment by mr337** [comment]
Gah, not able to reproduce it any more. Going to blame that on lack of sleep :)

**Comment by asyncee** [comment]
I have faced this issue today.
I'm using Linux Mint, and you are using MacOS (possibly).
You should note that file system in linux is case sensitive, but in MacOS it is not.

So, you are doing:

```
var DialogSlot = require('../components/DialogSlot');
```

instead of (this is the fix):

```
var DialogSlot = require('../components/dialogSlot');
```

because actual file is named with camel case style.

I just checked it - this fix works.

Should i do pull request?

**Comment by asyncee** [comment]
Sorry, did not noticed what you said about file name in latest pypi release, thats really strange - i have lektor 1.2.1 installed with bash script from getlektor site and the file name is `dialogSlot.jsx`, but in archive from pypi (1.2.1) it is `DialogSlot.jsx`.

This could happen if you re-uploaded fixed release 1.2.1 twice.
